src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from 'react';
+import { PeopleList } from '../../components/PeopleList/PeopleList';
+
+import { getPeople } from '../../api';
+import { Person } from '../../types/Person';
+import { Loader } from '../../components/Loader';
+import { ErrorType } from '../../types/ErrorType';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [isErrorLoading, setIsErrorLoading] = useState(false);
+
+  const isPeopleInServer = people.length > 0;
+
+  const loadPeople = () => {
+    setIsLoading(true);
+
+    getPeople()
+      .then((persons) => setPeople(persons))
+      .catch(() => setIsErrorLoading(true))
+      .finally(() => setIsLoading(false));
+  };
```

Comment:
It's better to move the function creation inside the `useEffect` where you use it.